fdbrpc/AsyncFileEIO.actor.h

```diff
@@ -246,6 +246,9 @@ class AsyncFileEIO : public IAsyncFile, public ReferenceCounted<AsyncFileEIO> {
 		if( flags & OPEN_READONLY )  oflags |= O_RDONLY;
 		if( flags & OPEN_READWRITE ) oflags |= O_RDWR;
 		if( flags & OPEN_ATOMIC_WRITE_AND_CREATE ) oflags |= O_TRUNC;
+#if defined(__linux__)
+		if ( flags & OPEN_UNBUFFERED ) oflags |= O_DIRECT;
```

Comment:
I think we need to offer a configuration where O_DIRECT is not used at all: see #842

Reply:
That's an interesting idea.  When I took a look at fuse for fault injection purpose, I was wondering the similar thing.  I think it's not a bad idea to have it as another knob.

Reply:
Apparently, fuse does support O_DIRECT.  But still, having a knob is not a bad idea, IMHO.
https://github.com/libfuse/libfuse/blob/44346a9885cb4567ea29c4cb089b1f249c0e46c2/NEWS#L128

Reply:
Agreed. I introduced a knob to disable O_DIRECT. If this is set this will also disable AIO (as AIO without O_DIRECT will always be blocking).